### PR TITLE
rules: clarify handling of late submissions

### PR DESCRIPTION
**Pull request by adilger**

### CODE REVIEW

**Review by gflofst — APPROVED** [review]

**Review by gmarkomanolis — DISMISSED** [review]

**Review by JulianKunkel — DISMISSED** [review]

**Review by gflofst — APPROVED** [review]

**Comment by adilger** [comment]
Dean, can you please review and approve this PR.

File: templates/Pages/rules_submission.php [hunk 1]
```diff
@@ -110,6 +110,16 @@
             "<span class="code">beegfs_ctl</span>"; however, additional
             subdirectories within these directories cannot be precreated.
         </li>
+        <li>
+            Submissions received after the posted deadline for a list may
+            only be accepted upon request to
+            <a href="mailto:committee@io500.org" class="link">committee@io500.org</a>.
```
Comment (seattleplus) [inline templates/Pages/rules_submission.php:116]: Should this be a , and not a . after the /a?

File: templates/Pages/rules_submission.php [hunk 2]
```diff
@@ -110,6 +110,16 @@
...
+            and only if the result does not affect the winner in any category
```
Comment (seattleplus) [inline templates/Pages/rules_submission.php:117]: This should be, "If the result..."

File: templates/Pages/rules_submission.php [hunk 3]
```diff
@@ -110,6 +110,16 @@
             "<span class="code">beegfs_ctl</span>"; however, additional
             subdirectories within these directories cannot be precreated.
         </li>
+        <li>
+            Submissions received after the posted deadline for a list may
```
Comment (seattleplus) [inline templates/Pages/rules_submission.php:114]: How about, "may be accepted (at the discussion of the io500 committee) up request to..."

File: templates/Pages/rules_submission.php [hunk 4]
```diff
@@ -110,6 +110,16 @@
...
+            only be accepted upon request to
+            <a href="mailto:committee@io500.org" class="link">committee@io500.org</a>.
+            and only if the result does not affect the winner in any category
+            after the official deadline. In the case where a late submission
+            would be the winner in any category, or is too near or after the
+            final list publication (at the discretion of the committee), then
```
Comment (seattleplus) [inline templates/Pages/rules_submission.php:120]: How about something like, " In the case where a late submission would be the winner in any category, then a key consideration for acceptance is ensuring there is enough time before publication of the final list.  Any submission not accepted for the current list would be automatically submitted to the following list.

**Review by gflofst — APPROVED** [review]

**Review by JulianKunkel — APPROVED** [review]

**Review by seattleplus — APPROVED** [review]

**Comment by gflofst** [comment]
@gmarkomanolis

**Review by gmarkomanolis — APPROVED** [review]

**Pull request merged**